templates/management-compose.ecs.yml

```diff
@@ -27,6 +27,8 @@ services:
       SAMPLE_BUCKET:
       SOLR_BASE_URL: http://${CLUSTER_NAME}-solr.${CLUSTER_NAME}:8983/solr
       POSTGRES_HOST: ${CLUSTER_NAME}-psql.${CLUSTER_NAME}
+    volumes:
+      - /brbl-dsu:/brbl-dsu
```

Comment:
I think this belongs only in worker-compose.ecs.yml (for the EC2 worker containers) not management-compose.ecs.yml (for the fargate management app).

Reply:
+1